package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "reify",
-  "version": "0.20.12",
+  "version": "0.20.13",
```

```suggestion
  "version": "0.20.12",
```
We'll do this on the `master` branch before publishing.